integration-cli/docker_cli_network_unix_test.go

```diff
@@ -1417,7 +1417,7 @@ func (s *DockerSuite) TestDockerNetworkInternalMode(c *check.C) {
 	c.Assert(waitRun("first"), check.IsNil)
 	dockerCmd(c, "run", "-d", "--net=internal", "--name=second", "busybox", "top")
 	c.Assert(waitRun("second"), check.IsNil)
-	_, _, err := dockerCmdWithTimeout(time.Second, "exec", "first", "ping", "-c", "1", "www.google.com")
+	_, _, err := dockerCmdWithError("exec", "first", "ping", "-W", "4", "-c", "1", "www.google.com")
 	c.Assert(err, check.NotNil)
```

Comment:
@thaJeztah  IMO,  we should do that. Just as we have done in other tests, we always check `c.Assert(err, check.NotNil)` first and then check the `err` message if it has the expected error.